### ISSUE TITLE
Rotation is break in the camera app

### ISSUE THREAD

**Issue by Aurnytoraink** [ubports/ubuntu-touch#1174]
- Device: Nexus 7 (2013 Wifi)
- Channel: Development
- Build: 20190704-32398
- Camera version: 3.1.0

### Steps to reproduce
Open the camera app, and take some picture

### Expected behavior
Basically, take a picture normally

### Actual behavior
Well when I take a picture in portrait mode the picture looks like [this](http://aurnytoraink.ddns.net/s/pnLCQNsqytsfc2g)
And when I'm in landscape mode (turned left) looks like [this](http://aurnytoraink.ddns.net/s/nQsGNYy4iFLRQXB) (which is normal).
But if you turned right it gets upside down.

Also, if you launched the app in landscape mode, the load screen looks like [this](http://aurnytoraink.ddns.net/s/6fxWLY8BwGPPRnw)

### Logfiles and additional information
No

**Comment by peat-psuwit** [comment]
First, I'm unable to open the linked images. Could you please attach them into the issue itself?
Second, could you please run [camera_orientation utility](https://11-192920238-gh.circle-artifacts.com/0/root/project/camera_orientation)? This will tell me how the system sees the camera.

**Comment by Flohack74** [comment]
@Aurnytoraink please fix the picture links, or better embed them here.

**Comment by Aurnytoraink** [comment]
Ok I update the link, using the imgur's service but that strange that didn't work because it's pointing to my Nextcloud instance

**Comment by Aurnytoraink** [comment]
Ok you right my server got some errors I need to fix it. Sorry I went offtopic

**Comment by Aurnytoraink** [comment]
@peat-psuwit Here is the result
![photo_2019-07-08_09-19-18](https://user-images.githubusercontent.com/28545669/60790433-ca555380-a161-11e9-9d61-c892cf02c828.jpg)

**Comment by peat-psuwit** [comment]
Could you try turning the device to landscape, turn it back to portrait and then take a photo?

**Comment by Aurnytoraink** [comment]
I tried but still missed up
![photo_2019-07-08_14-38-52](https://user-images.githubusercontent.com/28545669/60810816-3fd71900-a18e-11e9-8509-f0a992d46146.jpg)

**Comment by peat-psuwit** [comment]
Hmm, strange. Could you please:
- Enable developer mode in "System Settings > About > Developer mode"
- Install [this deb](https://ci.ubports.com/job/qtubuntu-camera/job/PR-4-head/1/artifact/cameraplugin-aal_0.4.1+ubports0+0~20190708124818.1~1.gbp7cedcd_armhf.deb). To install, run `sudo mount -o remount,rw /` followed by `sudo dpkg -i <deb file>`.
- Launch the camera app, take pictures in different directions, and grab the log from `~/.cache/upstart/application-click-com.ubuntu.camera_camera_3.1.0.log`. Please state which directions do you take your picture, as the log will contain 1 line per capture.

Also, does the viewfinder work correctly?

**Comment by Aurnytoraink** [comment]
Good luck
[application-click-com.ubuntu.camera_camera_3.1.0.log](https://github.com/ubports/ubuntu-touch/files/3368532/application-click-com.ubuntu.camera_camera_3.1.0.log)

**Comment by peat-psuwit** [comment]
How many times do you take your photos and in which directions? I have to correlate it with the log.

**Comment by Aurnytoraink** [comment]
Here you go
[test_photos.zip](https://github.com/ubports/ubuntu-touch/files/3368969/test_photos.zip)

**Comment by peat-psuwit** [comment]
Sorry to bother you again but **in which order** are the photos taken?

**Comment by Aurnytoraink** [comment]
@peat-psuwit Well, there are in the right order which mean:
- Back_L-L
- Back_L-R
- Back_P
- Back_P-I
- Front_L-L
- Front_L-R
- Front_P
- Front_P-I

**Comment by peat-psuwit** [comment]
Hmm, strange. From the log, it's indicated that the rotation is calculated correctly. However, it seems like it's not applied to the photo. If the logged rotation is applied to the photo, it'll be oriented correctly.

I'm now suspecting another package. Could you please run the following commands, reboot, and retest?

```sh
sudo mount -t tmpfs tmp /var/cache/apt/archives/
sudo apt update
sudo apt install libhybris=0.1.2+mariogrip2+0~20190615205852.12~1.gbpcf13ab libandroid-properties1=0.1.2+mariogrip2+0~20190615205852.12~1.gbpcf13ab libhybris-utils=0.1.2+mariogrip2+0~20190615205852.12~1.gbpcf13ab libhybris-test=0.1.2+mariogrip2+0~20190615205852.12~1.gbpcf13ab
```

**Comment by Aurnytoraink** [comment]
Hum doind `sudo apt update`, result with this

```
Ign:1 http://repo.ubports.com xenial InRelease
Ign:2 http://ports.ubuntu.com/ubuntu-ports xenial InRelease
Err:3 http://repo.ubports.com xenial Release       
  Could not open file /var/lib/apt/lists/partial/repo.ubports.com_dists_xenial_Release - open (30: Read-only file system) [IP: 104.24.1.9 80]
Ign:4 http://ports.ubuntu.com/ubuntu-ports xenial-updates InRelease
Ign:5 http://ports.ubuntu.com/ubuntu-ports xenial-security InRelease
Err:6 http://ports.ubuntu.com/ubuntu-ports xenial Release
  Could not open file /var/lib/apt/lists/partial/ports.ubuntu.com_ubuntu-ports_dists_xenial_Release - open (30: Read-only file system) [IP: 91.189.88.150 80]
Err:7 http://ports.ubuntu.com/ubuntu-ports xenial-updates Release
  Could not open file /var/lib/apt/lists/partial/ports.ubuntu.com_ubuntu-ports_dists_xenial-updates_Release - open (30: Read-only file system) [IP: 91.189.88.150 80]
Err:8 http://ports.ubuntu.com/ubuntu-ports xenial-security Release
  Could not open file /var/lib/apt/lists/partial/ports.ubuntu.com_ubuntu-ports_dists_xenial-security_Release - open (30: Read-only file system) [IP: 91.189.88.150 80]
Reading package lists... Done                      
W: chmod 0700 of directory /var/lib/apt/lists/partial failed - SetupAPTPartialDirectory (30: Read-only file system)
W: Not using locking for read only lock file /var/lib/apt/lists/lock
W: No sandbox user '_apt' on the system, can not drop privileges
E: The repository 'http://repo.ubports.com xenial Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ports.ubuntu.com/ubuntu-ports xenial Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ports.ubuntu.com/ubuntu-ports xenial-updates Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ports.ubuntu.com/ubuntu-ports xenial-security Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

```

**Comment by peat-psuwit** [comment]
Oh, sorry. I forgot to include read-write remount. Run `mount -o remount,rw /` and try again. (No need to repeat the first command)

**Comment by Aurnytoraink** [comment]
Well, it's continuing to do the same issue

**Comment by UniversalSuperBox** [comment]
On further investigation, this issue appears to be present in OTA-9 stable. So it is not a regression as such, and not quite as critical as previously believed. I'll remove the critical tag so it doesn't block further releases, but this is still a bug.

**Comment by Aurnytoraink** [comment]
Also, looks like the issue only happen in photo mode because when I take a video everything is okay

**Comment by dark-eye** [comment]
@Aurnytoraink Please make sure the datestamp on images option is turned off (it will also rotate the pictures due to EXIF data loss)

**Comment by Aurnytoraink** [comment]
@dark-eye It's already turned off

**Comment by Flohack74** [comment]
Is this fixed by OTA-10?

**Comment by UniversalSuperBox** [comment]
No, it is not. The Nexus 7's *taken* images are not rotated correctly, while the viewfinder is okay

**Comment by keitalbame** [comment]
I confirm that on OTA-10, I have the same rotate issue.

**Comment by lgtrombetta** [comment]
This issue is also present on the Nexus 6P.
